src/test/java/core/utils/WaiterUtils.java

```diff
@@ -0,0 +1,30 @@
+package core.utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Wait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WaiterUtils {
+
+    public static void waitElementToBeVisible(WebDriver driver, WebElement element) {
+        new WebDriverWait(driver, 30).until(ExpectedConditions.visibilityOf(element));
+    }
+
+    public static void waitForPageLoad(WebDriver driver) {
```

Comment:
why you need to pass driver here if it is singleton?

Reply:
because I use this driver in my method